packages/transformers/xml/src/XMLTransformer.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+import {JSDOM} from 'jsdom';
+import nullthrows from 'nullthrows';
+import semver from 'semver';
+
+export default (new Transformer({
+  canReuseAST({ast}) {
+    return ast.type === 'jsdom' && semver.satisfies(ast.version, '^16.6.0');
+  },
+
+  async parse({asset}) {
+    return {
+      type: 'jsdom',
+      version: '16.6.0',
+      program: new JSDOM(await asset.getBuffer(), {
```

Comment:
JSDOM seems a bit heavyweight for this use case. Should we use a simpler XML parser for this instead?